### PR TITLE
chore: Merge nv22 into master

### PR DESCRIPTION
**Pull request by rjan90**
## Proposed Changes
- Merge nv22 into master.

## Checklist

Before you mark the PR ready for review, please make sure that:

- [x] Commits have a clear commit message.
- [x] PR title is in the form of of `<PR type>: <area>: <change being made>`
  - example: ` fix: mempool: Introduce a cache for valid signatures`
  - `PR type`: fix, feat, build, chore, ci, docs, perf, refactor, revert, style, test
  - `area`, e.g. api, chain, state, market, mempool, multisig, networking, paych, proving, sealing, wallet, deps
- [ ] If the PR affects users (e.g., new feature, bug fix, system requirements change), update the CHANGELOG.md and add details to the UNRELEASED section.
- [ ] New features have usage guidelines and / or documentation updates in
  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

File: build/version.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ func BuildTypeString() string {
 }
 
 // BuildVersion is the local build version
-const BuildVersion = "1.25.3-dev"
+const BuildVersion = "1.26.0-rc2"
```
Comment (arajasek) [inline build/version.go:40]: I don't think we want this, should be 1.27.0-dev, I think.

Comment (rjan90) [reply build/version.go]: Updated in https://github.com/filecoin-project/lotus/pull/11699/commits/a613651b3bf49e456d9e4a7381157b7409e9022c

File: cli/filplus.go [hunk 2]
```diff
@@ -427,7 +427,7 @@ var filplusListClaimsCmd = &cli.Command{
 			var claimList []map[string]interface{}
 
 			for key, val := range claims {
-				if ts.Height() > claim.TermMax+claim.TermStart || !cctx.IsSet("expired") {
+				if tsHeight > val.TermMax || !expired {
```
Comment (arajasek) [inline cli/filplus.go:430]: @LexLuthr I think this should be `tsHeight > val.TermStart + val.TermMax`, @LexLuthr can confirm.

Comment (LexLuthr) [reply cli/filplus.go:430]: @arajasek You are right. This should be val.TermStart + val.TermMax to filter out the expired claims.

File: node/impl/full/dummy.go [hunk 3]
```diff
@@ -188,3 +189,17 @@ func (e *EthModuleDummy) EthTraceReplayBlockTransactions(ctx context.Context, bl
 
 var _ EthModuleAPI = &EthModuleDummy{}
 var _ EthEventAPI = &EthModuleDummy{}
+
+var ErrActorEventModuleDisabled = errors.New("module disabled, enable with Fevm.EnableActorEventsAPI")
```
Comment (magik6k) [inline node/impl/full/dummy.go:193]:
```suggestion
var ErrActorEventModuleDisabled = errors.New("module disabled, enable with Events.EnableActorEventsAPI")
```

**Review by magik6k — COMMENTED** [review]
Other than the one nit and the definitely needed https://github.com/filecoin-project/lotus/pull/11704 this looks good

**Pull request merged**